source/fundamentals/connection/connection-options.txt

```diff
@@ -206,6 +206,11 @@ parameters of the connection URI to specify the behavior of the client.
      - boolean
      - Specifies that the driver must connect to the host directly.
 
+   * - **maxConnecting**
+     - integer
+     - Specifies the number of connections the driver concurrently
+       establishes before making them available in the connection pool.
```

Comment:
Suggestion:

> Specifies the number of concurrent connections the driver establishes before making them available in the connection pool.

The adverb "concurrently" should describe the connections not how the driver establishes them.